### PR TITLE
[issue#602] Enable the popup window only if the user has typed in a keyword.

### PR DESCRIPTION
**Pull request by dongchime**
**- Enable the popup window only if the user has typed in a keyword.**

ezgif com-video-to-gif

### CODE REVIEW

File: examples/twittermap/web/public/javascripts/map/countmap.js [hunk 1]
```diff
@@ -499,15 +501,15 @@ angular.module('cloudberry.map')
...
-          // bind a pop up window
-          $scope.popUp = L.popup({autoPan:false});
-          layer.bindPopup($scope.popUp).openPopup();
-          $scope.popUp.setContent(getPopupContent()).setLatLng([$scope.selectedPlace.properties.popUpLat,$scope.selectedPlace.properties.popUpLog]);
-
-          addPopupEvent();
-          drawLineChart();
-
+          if ($scope.checkIfQueryIsRequested == true) {
+              // bind a pop up window
```
Comment (chenlica) [inline examples/twittermap/web/public/javascripts/map/countmap.js]: Avoid this type of unnecessary format changes that are not related to the logic.

**Comment by baiqiushi** [comment]
Please resolve the conflicts. @dongchime
